com.sap.cloud.lm.sl.cf.web/manifests/logs

```diff
@@ -0,0 +1,1523 @@
+Retrieving logs for app deploy-service in org deploy-service / space i500571 as i500571...
```

Comment:
Be careful about including such files in your commits. Always check what you've included in the commit before pushing it. Use `gitk` or something similar for this purpose.

Reply:
Also, remove the `cli.mtar` file from this commit.